lib/storage/metadata/mongoclient/MongoClientInterface.js

```diff
@@ -412,6 +412,72 @@ class MongoClientInterface {
         });
     }
 
+    /**
+     *
+     * @param {String} bucketName - name of bucket
+     * @param {String} capabilityName - name of capability
+     * @param {String} [capabilityField] - name of capability field
+     * @param {Object} capability - capability object
+     * @param {Sbject} log - logger
+     * @param {Function} cb - callback
+     * @return {undefined}
+     */
+    putBucketAttributesCapabilities(bucketName, capabilityName, capabilityField, capability, log, cb) {
```

Comment:
For my understanding, why are capabilityName and capabilityField separate fields from the capability object? Could they not be recovered from that object to avoid these extra parameters?

Reply:
It comes from the fact that wa have both `VeeamSOSAPI` and two sub fields, one for each XML file. If we update the whole `VeeamSOSAPI`, it won't fix the concurrency issue: we want to be able to set a unique "file" for any identity, so we need two layers. We could do it a bit more dynamically, but to me it'll reduce the readability...